_topic_map.yml

```diff
@@ -736,7 +747,7 @@ Topics:
 #---
 Name: Images
 Dir: openshift_images
-Distros: openshift-enterprise,openshift-webscale,openshift-origin,openshift-dedicated,openshift-online,openshift-aro
+Distros: openshift-enterprise,openshift-webscale,openshift-origin,openshift-dedicated,openshift-online,openshift-aro,openshift-moa
```

Comment:
@lamek replace this with openshift-* since all distros are using it. 

Same for anywhere else where all distros are used.

Reply:
:+1: Only 9 places I saw. I replaced them all.